stripe-core/src/main/java/com/stripe/android/core/networking/AnalyticsRequest.kt

```diff
@@ -29,5 +32,27 @@ data class AnalyticsRequest(
 
     internal companion object {
         internal const val HOST = "https://q.stripe.com"
+
+        fun create(
+            params: Map<String, *>,
+            headers: Map<String, String>,
+        ): AnalyticsRequest {
```

Comment:
Adding this converter method so that we don’t have to change all the call sites.